src/journal/journalctl.c

```diff
@@ -2621,8 +2621,8 @@ int main(int argc, char *argv[]) {
                                 if (r >= 0) {
                                         if (previous_boot_id_valid &&
                                             !sd_id128_equal(boot_id, previous_boot_id))
-                                                printf("%s-- Reboot --%s\n",
-                                                       ansi_highlight(), ansi_normal());
+                                                printf("%s-- Reboot "SD_ID128_FORMAT_STR" --%s\n",
+                                                       ansi_highlight(), SD_ID128_FORMAT_VAL(boot_id), ansi_normal());
```

Comment:
I think this is nice, but I think just the ID without context is unclear. What about adding some prefix like "boot-id=..." ?
(Esp. "Reboot foo" suggests that this number is somehow about the reboot.)

Reply:
Hmm, maybe rename "Reboot" to "Boot" ?

Reply:
That's a pretty good idea.

I tried a few more formats and I think I like just "-- Boot deadbeef123456789 --" the best.